Persist volume, mute, and playback speed across sessions (#872)
bitmovin/bitmovin-player-ui: CHANGELOG.md

### Added

> - `UIPreferences`: persists volume, mute state, and playback speed across sessions and reapplies them on player init (after `Ready`). Configurable via `UIConfig.enablePersistentPreferences` (default `true`). All persistent storage routes through the existing `disableStorageApi` flag and degrades to a no-op when `localStorage` is unavailable (private browsing, restricted WebViews, RDK / set-top boxes).

Reviewer: We are trying to write the changelog entries as sentences including the heading term:

```suggestion
- `UIConfig.enablePersistentPreferences` to configure whether the UI persists volume, mute state, and playback speed across sessions and reapplies them on player init.
```

> - `UIConfig.enablePersistentPreferences` and `UIConfig.showPersistentPreferencesToggle` to configure whether the UI persists volume, mute state, and playback speed across sessions and reapplies them on player init — automatically and/or via an end-user opt-in toggle in the settings panel. Both default to `false`; they have no effect when `disableStorageApi` is set or `localStorage` is unavailable (private browsing, restricted WebViews, RDK / set-top boxes).

Reviewer: Please merge development and move the entry to the unreleased section.

> - `UIConfig.enablePersistentPreferences` and `UIConfig.showPersistentPreferencesToggle` to configure whether the UI persists volume, mute state, and playback speed across sessions and reapplies them on player init — automatically and/or via an end-user opt-in toggle in the settings panel. Both default to `false`; they have no effect when `disableStorageApi` is set or `localStorage` is unavailable (private browsing, restricted WebViews, RDK / set-top boxes).

Reviewer: Nit: this entry is very detailed. feels like parts of it should be in the documentation and not in the changelog itself.

```suggestion
- `UIConfig.enablePersistentPreferences` and `UIConfig.showPersistentPreferencesToggle` to configure whether the UI persists volume, mute state, and playback speed across sessions.
```
